tests/common/system_utils/docker.py

```diff
@@ -158,8 +158,7 @@ def swap_syncd(dut):
               "{}/{}".format(registry.host, docker_rpc_image),
               sonic_version)
 
-    dut.command("systemctl reset-failed swss")
-    dut.command("systemctl start swss")
+    dut.command("config reload -y")
```

Comment:
Please use helper function config_reload as this example: https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/test_sequential_restart.py#L33

config reload takes time to get dut ready. Without proper wait, the next test could find system not in healthy state. Please change config reload below too.

Reply:
Neat! Updated the PR.